lib/netsuite/records/item_vendor.rb

```diff
@@ -0,0 +1,18 @@
+module NetSuite
+  module Records
+    class ItemVendor
+      include Support::Fields
+      include Support::Records
+      include Namespaces::ListAcct
+
+      fields :vendor, :purchase_price, :preferred_vendor
+
+      attr_reader :internal_id
+
+      def initialize(attributes = {})
+        @internal_id = attributes.delete(:internal_id) || attributes.delete(:@internal_id)
```

Comment:
@rudyyazdi does the ItemVendor have a `internalId`? I'm not seeing one in the schema, but sometimes NS will return XML elements that don't exist in the schema

https://system.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2015_1/schema/other/itemvendor.html?mode=package